packages/datepicker/src/utils/platform.native.ts

```diff
@@ -0,0 +1,2 @@
+export const isWeb = false;
+export const isNative = true;
```

Comment:
How will it work, react automatically will pick `platform.native.ts` or `platform.ts` based on ReactNavite or not?
Since this is buildable package I can't see how it will work, (I'm not an expert in React Native).

Reply:
yeah bundlers will handle that file. and builded artifact also include that.
so on react native it only resolves .native file

Reply:
I thought we need to create a separate build for the React Native